### PR TITLE
[#1839]add ContentType return while getConfig

### PR DESCRIPTION
**Pull request by wangweizZZ**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

#1839

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by zongtanghu** [comment]
The CI test cases has failed, please check it.@wangwei

**Comment by wangweizZZ** [comment]
ci had passed .please help review this pr @zongtanghu @KomachiSion

**Comment by zongtanghu** [comment]
Please help to review this pr. @KomachiSion @chuntaojun

File: config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigServletInner.java [hunk 1]
```diff
@@ -136,8 +138,18 @@ public String doGetConfig(HttpServletRequest request, HttpServletResponse respon
                             isBeta = true;
                         }
                     }
-                    String configType = cacheItem.getType();
-                    response.setHeader("Config-Type", (null != configType) ? configType : "text");
+                    
+                    final String configType =
```
Comment (chuntaojun) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigServletInner.java:142]: 去除三元表达式

Comment (KomachiSion) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigServletInner.java:142]: 我看着没什么问题啊，这里二元表达式挺好的，原本也是二元表达式。

Comment (wangweizZZ) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigServletInner.java:142]: @zongtanghu 你觉得呢？

**Pull request merged**